test/screens/v2/candidate_generator/dup/departures_test.exs

```diff
@@ -59,7 +59,12 @@ defmodule Screens.V2.CandidateGenerator.Dup.DeparturesTest do
          [
            %Departure{
              prediction:
-               struct(Prediction, id: "A", route: %Route{id: "Test"}, stop: struct(Stop))
+               struct(Prediction,
+                 id: "A",
+                 route: %Route{id: "Test"},
+                 stop: struct(Stop),
+                 trip: struct(Trip)
```

Comment:
Is there a reason why you needed to add `trip` fields to all of these test values? I don't see a direct relation to the code changes above.